Automattic/newspack-theme
# fix: correct related events header font

### All Submissions:


### Changes proposed in this Pull Request:

This small edit makes sure the 'Related Events' header, displayed by The Events Calendar PRO plugin, to make sure it's picking up the theme's and/or site's custom header font.

### How to test the changes in this Pull Request:

1. Start with a test site with the Events Calendar PRO plugin set up, and a few events added (I think if you set a couple events to use the same venue, they should show as related - I think that's why they're showing for me at least!). Make sure under your settings that:
     * AMP is turned off for these pages
     * That you're running the v2 views (under WP Admin > Events > Settings > Display, the Enable updated designs for all calendar views option should be checked).
     * That your site is set to use 'Tribe Events Styles' (under WP Admin > Events > Settings > Display)
     * That your site is set to use the 'Default Events Template' (under WP Admin > Events > Settings > Display).
2. View a single event and note that, regardless of your theme or custom typography settings, the "Related Events" text uses Helvetica Neue:

![image](https://user-images.githubusercontent.com/177561/127708996-e9cd1848-eaed-47ab-b808-fdaae5e9e657.png)

3. Apply the PR and run `npm run build`.
4. Confirm that the header is using either your theme's header font, or your custom font if set:

![image](https://user-images.githubusercontent.com/177561/127708924-836502cc-5766-4804-adb7-2470275931ed.png)

### Other information:
